### PR TITLE
Replaced TOC markers in MD files

### PR DESCRIPTION
**Pull request by Vedanshu7**
#736 
Replaced TOC markers in MD files. 
Create a sh script that will replace old markers, if any, with new ones with the use of dev_scripts/replace_text.py.
Run the script in only the docs directory because the files that have old markers, are only present in the docs folder.

### CODE REVIEW

File: docs/coding/all.integrate_repos.how_to_guide.md [hunk 1]
```diff
@@ -10,7 +10,7 @@
   - [Double-check the integration](#double-check-the-integration)
   - [Run tests](#run-tests)
 
-<!--ts-->
+<!-- toc -->
```
Comment (gpsaggese) [inline docs/coding/all.integrate_repos.how_to_guide.md:13]: -> tocstop

File: docs/kaizenflow/all.run_end_to_end_Mock2_system.tutorial.md [hunk 2]
```diff
@@ -271,10 +272,12 @@ Data snippet:
 
 ```markdown
                                      start_datetime              timestamp_db     open     high      low    close  volume  asset_id
-end_datetime
-2023-08-08 15:00:00-04:00 2023-08-08 14:00:00-04:00 2023-08-08 15:00:10-04:00   985.34   986.36   984.87   986.25   935.0         0
-2023-08-08 15:00:00-04:00 2023-08-08 14:00:00-04:00 2023-08-08 15:00:10-04:00  1005.38  1006.65  1005.07  1006.34   948.0         1
-2023-08-08 15:00:00-04:00 2023-08-08 14:00:00-04:00 2023-08-08 15:00:10-04:00  1002.44  1002.49  1001.60  1001.88  1013.0         2
+
```
Comment (gpsaggese) [inline docs/kaizenflow/all.run_end_to_end_Mock2_system.tutorial.md:275]: The linter is formatting this incorrectly.

Does using ```verbatim here solves the problem?
Can you revert this and retry?

File: docs/kaizenflow/all.run_end_to_end_Mock2_system.tutorial.md [hunk 3]
```diff
@@ -376,34 +379,72 @@ When executing the `ProcessForecastsNode` the System:
 
 ```markdown
 # last target positions=
```
Comment (gpsaggese) [inline docs/kaizenflow/all.run_end_to_end_Mock2_system.tutorial.md:381]: Same here and everywhere

**Review by gpsaggese — COMMENTED** [review]
Going in the right direction. Some suggestions

File: docs/coding/all.integrate_repos.how_to_guide.md (hunk 1 above)
Comment (Vedanshu7) [reply docs/coding/all.integrate_repos.how_to_guide.md]: Resolved.

File: docs/kaizenflow/all.run_end_to_end_Mock2_system.tutorial.md (hunk 2 above)
Comment (Vedanshu7) [reply docs/kaizenflow/all.run_end_to_end_Mock2_system.tutorial.md]: Resolved.

File: docs/kaizenflow/all.run_end_to_end_Mock2_system.tutorial.md (hunk 3 above)
Comment (Vedanshu7) [reply docs/kaizenflow/all.run_end_to_end_Mock2_system.tutorial.md:379]: Resolved.

**Review by sonaalKant — APPROVED** [review]
Great Job! LGTM.

**Review by gpsaggese — COMMENTED** [review]
Good work

**Pull request merged**